code/asteroid/asteroid.cpp

```diff
@@ -599,7 +573,7 @@ void asteroid_create_all()
 		} else {
 			Assert(num_debris_types > 0);
 
-			int rand_choice = rand() % max_weighted_range;
```

Comment:
Is the changed random generator still going to produce the same results?

Reply:
Yes, it was merely changed to dealing with floats instead of ints for ease of use by modders, and should have the same behavior.